frontend/tests/unit/components/shasIntake/SubmissionIntakeForm.spec.ts

```diff
@@ -144,18 +144,17 @@ describe('SubmissionIntakeForm', () => {
   });
 
   it('checks categories for valid data', async () => {
-    const applicantTest = submissionIntakeSchema.validateAt('contacts', {
-      contacts: [
-        {
-          firstName: 'testFirstName',
-          lastName: 'testLastName',
-          phoneNumber: '2501234567',
-          email: 'test@test.com',
-          contactApplicantRelationship: ProjectRelationship.AGENT,
-          contactPreference: ContactPreference.PHONE_CALL
-        }
-      ]
-    });
+    // Contacts are kinda whack right now
```

Comment:
reason to keep?

Reply:
It should be there, but the way contacts are currently being handled in the forms currently makes validating that one bit a pain. This needs to go back in at some point.